### DISCUSSION TITLE
`VIRTUAL_ENV` variable accessible in `settings.json`

### DISCUSSION THREAD

**Discussion by DetachHead** [microsoft/pylance-release#2632] [Enhancement]
i want to be able to do this:
```json
{
    "python.analysis.typeshedPaths": ["${env:VIRTUAL_ENV}/lib/site-packages/mypy/typeshed"]
}
```
but i can't because i think the `VIRTUAL_ENV` variable is only accessible within the virtual env. ideally i would be able to use `${command:python.interpreterPath}` but i understand those commands don't work in `settings.json`

see https://github.com/microsoft/pylance-release/issues/2221#issuecomment-1007045231

**Comment by judej** [comment]
Thanks for the request. This is not something that is currently supported, but will reevaluate based on upvotes or community feedback.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
